llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -26060,22 +26060,43 @@ SDValue X86TargetLowering::LowerINTRINSIC_WO_CHAIN(SDValue Op,
       if (CC == ISD::SETLT || CC == ISD::SETLE)
         std::swap(LHS, RHS);
 
-      SDValue Comi = DAG.getNode(IntrData->Opc0, dl, MVT::i32, LHS, RHS);
+      // For AVX10.2, Support EQ and NE
```

Comment:
Add the ending `.` for comments. The same below.

Reply:
done